docs/source/guides/logging.rst

```diff
@@ -2,94 +2,92 @@
 Customize Logging
 =================
 
-BentoML provides a powerful and detailed logging pattern out of the box. Request logs for webservices are logged along with requests to each of the model runner services. We use `RichHandler <https://rich.readthedocs.io/en/stable/logging.html>`_ to provide appropriate color coding to the logs in order to make them more easily read.
+BentoML provides a powerful and detailed logging pattern out of the box. Request logs for
```

Comment:
Should we add a section here about logging out side of the context of model server? like giving user an example of configuring logging before calling `save_model`, similar to this example https://github.com/bentoml/gallery/blob/main/quickstart/train.py#L4-L6